libs/entitlements-runtime/src/main/java/org/elasticsearch/entitlements/runtime/api/EntitlementChecks.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+package org.elasticsearch.entitlements.runtime.api;
+
+public class EntitlementChecks {
+    static boolean agentSaidHello = false;
+
+    public static void helloFromAgent() {
```

Comment:
can we make this not public? It should be an impl detail?

I also would prefer it not be "hello" (sounds like a toy project), but something that can go into production like "agent booted".

Reply:
If I selectively export it to the agent, then public is ok, right?